.circleci/config.yml

```diff
@@ -107,21 +108,23 @@ jobs:
           path: /go/src/github.com/Azure/acs-engine/_logs
       - store_artifacts:
           path: /go/src/github.com/Azure/acs-engine/_output
-  k8s-1.9-release-e2e:
+  k8s-1.10-release-e2e:
     <<: *defaults
     steps:
       - checkout
       - run: |
           echo 'export TIMEOUT=20m' >> $BASH_ENV
-          echo 'export ORCHESTRATOR_RELEASE=1.9' >> $BASH_ENV
+          echo 'export ORCHESTRATOR_RELEASE=1.10' >> $BASH_ENV
           echo 'export CLUSTER_DEFINITION=examples/e2e-tests/kubernetes/release/default/definition.json' >> $BASH_ENV
           echo 'export CREATE_VNET=true' >> $BASH_ENV
+          echo 'export ENABLE_KMS_ENCRYPTION=true' >> $BASH_ENV
```

Comment:
should this be enabled for 1.12 as well?

Reply:
this diff is hard to read, but yeah, I am moving forward 1.11's test config to 1.12 (which includes KMS encryption)

Reply:
Got it, I think I got mixed up in the diff